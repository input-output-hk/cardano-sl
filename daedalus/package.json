{
  "name": "daedalus-client-api",
  "version": "0.0.1",
  "description": "Daedalus wallet client library",
  "main": "output/Daedalus.ClientApi/index.js",
  "scripts": {
    "start": "npm run build:dev",
    "preinstall": "type -P psc || type -P purs || npm install purescript@0.11.5",
    "postinstall": "bower cache clean && bower install",
    "clean": "rimraf dist && rimraf output",
    "build:prod": "npm run clean && mkdir dist && cross-env NODE_ENV=prod ./node_modules/.bin/webpack --config webpack.config.babel.js --progress && cross-env NODE_ENV=prod ./node_modules/.bin/webpack --config webpack.config.babel.js --progress --bail",
    "build:dev": "rimraf output && cross-env NODE_ENV=dev ./node_modules/.bin/webpack-dev-server --config webpack.config.babel.js --progress && cross-env NODE_ENV=dev ./node_modules/.bin/webpack-dev-server --config webpack.config.babel.js --progress --bail",
    "test": "npm run build:prod && npm run test:mocha",
    "test:mocha": "mocha ./test/**/*.test.js"
  },
  "keywords": [
    "daedalus",
    "cryptocurrency",
    "avm",
    "cardano",
    "wallet"
  ],
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "babel-core": "^6.26.0",
    "babel-loader": "^7.1.1",
    "babel-plugin-transform-es2015-destructuring": "^6.23.0",
    "babel-preset-es2015": "^6.24.1",
    "babel-preset-stage-2": "^6.24.1",
    "bower": "^1.8.0",
    "chai": "^4.1.1",
    "cross-env": "^5.0.5",
    "extract-text-webpack-plugin": "^3.0.0",
    "html-webpack-plugin": "^2.30.1",
    "mocha": "^3.5.0",
    "nock": "^9.0.14",
    "pulp": "^11.0.2",
    "purescript-psa": "^0.5.1",
    "purs-loader": "^3.1.0",
    "rimraf": "^2.6.1",
    "sinon": "^3.2.1",
    "text-encoding": "^0.6.4",
    "webpack": "3.5.5",
    "webpack-dev-server": "2.7.1",
    "xhr2": "^0.1.4"
  },
  "//": "NOTE: if you will be bumping bip39 to >=2.2.0 be aware of https://issues.serokell.io/issue/VD-95 . In this case you will have to modify how we validate paperVendMnemonics. Affected endpoints: isValidMnemonic, redeemAdaPaperVend",
  "dependencies": {
    "bip39": "2.2.0",
    "blakejs": "^1.1.0",
    "bs58": "^4.0.1",
    "json-loader": "^0.5.7",
    "ws": "^3.1.0"
  }
}
