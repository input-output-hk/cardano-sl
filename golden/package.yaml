name:                golden
version:             0.1.0.0
license:             BSD3
author:              "IOHK golden tests"
maintainer:          "example@example.com"
copyright:           "2018 IOHK"

dependencies:
- base >= 1 && < 5
- foundation
- basement
- bytestring
- memory
- inspector
- cryptonite
- cardano-crypto
- cardano-sl-crypto
- cardano-sl-binary
- cardano-sl-core

tests:
  golden-test:
    main:                Spec.hs
    source-dirs:         test
    ghc-options:
    - -threaded
    - -rtsopts
    - -with-rtsopts=-N
